app/frontend/src/features/map/MapPage.tsx

```diff
@@ -1,41 +1,50 @@
 import * as React from "react";
+import { makeStyles } from "@material-ui/core";
 import PageTitle from "../../components/PageTitle";
 import Map from "../../components/Map";
 import { userRoute } from "../../AppRoutes";
 import { LngLat, Map as MapboxMap } from "mapbox-gl";
 import { useHistory, useLocation } from "react-router-dom";
+import classNames from "classnames";
 
-import { addClusteredUsersToMap } from "./ClusteredUsers";
+import { addClusteredUsersToMap } from "./clusteredUsers";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    maxWidth: "100vw",
+    height: "80vh",
+    border: "1px solid black",
+  },
+}));
 
 export default function MapPage() {
   const history = useHistory();
 
   const location = useLocation();
 
-  const userClicked = (ev: any) => {
-    const username = ev.features[0].properties.username
+  const classes = useStyles();
+
+  const handleClick = (ev: any) => {
+    const username = ev.features[0].properties.username;
     history.push(`${userRoute}/${username}`, location.state);
   };
 
   const initializeMap = (map: MapboxMap) => {
     map.on("load", () => {
-      addClusteredUsersToMap(map, userClicked);
+      addClusteredUsersToMap(map, handleClick);
     });
   };
 
   return (
     <>
       <PageTitle>MapPage</PageTitle>
-      <div
-        style={{ maxWidth: "100vw", height: "80vh", border: "1px solid black" }}
-      >
-        <Map
-          initialZoom={1}
-          initialCenter={new LngLat(0, 0)}
-          grow
-          postMapInitialize={initializeMap}
-        />
-      </div>
+      <Map
+        initialZoom={1}
+        initialCenter={new LngLat(0, 0)}
+        grow
+        postMapInitialize={initializeMap}
+        className={classNames(classes.root)}
```

Comment:
you don't actually need `classNames` for just one class (take out the import too if you change this)
```suggestion
        className={classes.root}
```